go/src/meguca/db/util.go

```diff
@@ -124,6 +124,24 @@ func StartTransaction() (*sql.Tx, error) {
 	return db.Begin()
 }
 
+// GetPcount retrieves current pyu counter
+func GetPcount() (c uint64, err error) {
+	err = prepared["get_pcount"].QueryRow().Scan(&c)
+	return
+}
+
+// IncrementPcount increments the pyu counter by one and returns the new counter
+func IncrementPcount() (c uint64, err error) {
+	err = prepared["increment_pcount"].QueryRow().Scan(&c)
+	return
+}
+
+// SetPcount sets the pyu counter. Only used in tests.
+func SetPcount(c uint64) error {
+	_, err := db.Exec(`UPDATE boards SET pcount = $1`, c)
+	return err
+}
+
```

Comment:
These update and read ALL the boards at the same time.

Reply:
Yeah, I only realized this a bit before you mentioned it.
I think just creating a different table as (id (board), pcount) would work out.

Reply:
There is a problem I'm having however, and that's the fact that I can't get db.Exec to do more than 1 input, I need 2 for the board ID as well as the uint64.

Reply:
Actually never mind, I can just reuse the sql from before you killed pyu.

Reply:
Oh man this is amazing, I spent 5 hours on this problem trying all kinds of crazy things, even getting pretty heavy into the squirrel sql library, and it turned out all I had to do to get `WHERE` working was add the board ID to `QueryRow()` and some other minor alterations.

Reply:
You could always just check how the other db files do things.